deploy/charts/cert-manager/values.yaml

```diff
@@ -195,6 +195,12 @@ webhook:
   # Optional additional annotations to add to the webhook Pods
   # podAnnotations: {}
 
+  # Optional additional annotations to add to the webhook MutatingWebhookConfiguration
+  # mutatingWebhookAnnotations: {}
```

Comment:
I'd call this `mutatingWebhookConfigurationAnnotations`, to be consistent with the other annotation fields which follow the pattern `<resource_kind>Annotations:`

Reply:
Thx. Resolved and force-pushed.